hopsworks-common/src/main/java/io/hops/hopsworks/common/serving/inference/logger/KafkaInferenceLog.java

```diff
@@ -35,11 +35,11 @@
   public KafkaInferenceLog() {
   }
 
-  public KafkaInferenceLog(TfServing tfServing, String inferenceRequest,
+  public KafkaInferenceLog(Serving serving, String inferenceRequest,
```

Comment:
I think here we should log also the model type.

Reply:
Done